sidebars.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
     "📚 Playbooks": [
       "playbooks/browse",
       "playbooks/how-to-make-it-without-technical-skills",
-      "playbooks/coaching-eth",
+      "playbooks/how-to-coach-eth",
```

Comment:
@chair28980 I just checked your last update and all good but I noticed there's a conflict in `sidebars.js`. once that's sorted, it's good to go. 🙏🏻

Reply:
GAHH WHY IS THIS HAPPENING AHHHH

Tyty will get things fixed and ping when ready.

Reply:
GAHH WHY IS THIS HAPPENING AHHHH

Tyty will get things fixed and ping when ready.

Reply:
> GAHH WHY IS THIS HAPPENING AHHHH
😅 I can relate. I responded to your DM on Discord.